src/features/admin/survey-administrations/[id]/view-survey-admin-header.tsx

```diff
@@ -182,11 +182,9 @@ export const ViewSurveyAdminHeader = () => {
             />
           </div>
           <div className='flex justify-between gap-4'>
-            {survey_administration?.status === SurveyAdministrationStatus.Closed && (
-              <Button size={isMobile ? "small" : "medium"} onClick={handleViewResults}>
-                <div className='whitespace-nowrap'>Results</div>
-              </Button>
-            )}
+            <Button size={isMobile ? "small" : "medium"} onClick={handleViewResults}>
```

Comment:
@emmaacha i think available lang sha if Ongoing and Closed?

Reply:
Done ^^